### PR TITLE
OLMIS-7592:Proof of delivery - adding user's signature

### PR DESCRIPTION
**Pull request by mjozefow**
Adding user's signature after submitting PoD.
Jacoco Test Report plugin has been modified and now access to field is via setter.
Updated changelog with new improvements.

### CODE REVIEW

File: src/main/java/org/openlmis/fulfillment/web/util/StockEventBuilder.java [hunk 1]
```diff
@@ -255,4 +255,8 @@ private UUID getNodeId(UUID fromFacilityId, UUID toFacilityId, UUID programId,
     throw new ValidationException(EVENT_MISSING_SOURCE_DESTINATION, toFacility.getCode());
   }
 
+  private String getUserSignature() {
+    return authenticationHelper.getCurrentUser().getFirstName() + " " +
```
Comment (kmadej) [inline src/main/java/org/openlmis/fulfillment/web/util/StockEventBuilder.java:259]: use StringBuilder

Comment (kmadej) [reply src/main/java/org/openlmis/fulfillment/web/util/StockEventBuilder.java:259]: name method differently, you are building/creating user signature, not getting

Comment (kmadej) [reply src/main/java/org/openlmis/fulfillment/web/util/StockEventBuilder.java:259]: do not retrieve current user 3 times in the method, performance problem

Comment (kmadej) [reply src/main/java/org/openlmis/fulfillment/web/util/StockEventBuilder.java:259]: instead of " " try to use constant from util classes

Comment (kmadej) [reply src/main/java/org/openlmis/fulfillment/web/util/StockEventBuilder.java:259]: what if firstNAme or lastNAme is null ?

**Review by kmadej — CHANGES_REQUESTED** [review]

File: src/main/java/org/openlmis/fulfillment/web/util/StockEventBuilder.java (hunk 1 above)
Comment (mjozefow) [reply src/main/java/org/openlmis/fulfillment/web/util/StockEventBuilder.java]: > use StringBuilder

Fixed

Comment (mjozefow) [reply src/main/java/org/openlmis/fulfillment/web/util/StockEventBuilder.java]: All reported comments has been fixed

**Review by kmadej — APPROVED** [review]
looks good to me

**Review by NikolaLaskowska — APPROVED** [review]

**Pull request merged**